src/chromium/interfaces/listeners.js

```diff
@@ -19,7 +41,27 @@ export function initPlatformListeners() {
   });
 
   browser.runtime.onInstalled.addListener(async () => {
+    registerEventRules();
     await getIsFirefoxInstalled();
     browser.storage.sync.set({ currentExternalBrowser: "Firefox" });
   });
+
+  // From: https://github.com/Rob--W/crxviewer/blob/942b02fa871ff4ef6a29cf18b266711725314e86/src/background.js#L325-L341
+
+  //// Work-around for crbug.com/388231: onInstalled is not fired when the
+  //// extension was disabled during an update.
+  browser.runtime.onStartup.addListener(() => {
+    registerEventRules();
+  });
+
+  //// Work-around for crbug.com/264963: onInstalled is not fired when the
+  //// extension is run in incognito mode. Although not documented, incognito
+  //// contexts have their own declarativeContent rule store.
+  if (browser.extension.inIncognitoContext) {
```

Comment:
Your extension does not have `"incognito":"split"` in manifest.json, so this condition is never reached. You may as well delete this whole condition+block and comment.